instrumentation/aws-sdk/aws-sdk-2.2/library/src/testBedrockRuntime/java/io/opentelemetry/instrumentation/awssdk/v2_2/internal/Aws2BedrockRuntimeTest.java

```diff
@@ -858,4 +868,1842 @@ private static ToolConfiguration currentWeatherToolConfig() {
+                        .putDocument(
+                            "toolSpec",
+                            Document.mapBuilder()
+                                .putString("name", "get_current_weather")
+                                .putString(
+                                    "description", "Get the current weather in a given location.")
+                                .putDocument(
+                                    "inputSchema",
+                                    Document.mapBuilder()
+                                        .putDocument(
+                                            "json",
+                                            Document.mapBuilder()
+                                                .putString("type", "object")
+                                                .putDocument(
+                                                    "properties",
+                                                    Document.mapBuilder()
+                                                        .putDocument(
+                                                            "location",
+                                                            Document.mapBuilder()
+                                                                .putString("type", "string")
+                                                                .putString(
+                                                                    "description",
+                                                                    "The name of the city")
+                                                                .build())
+                                                        .build())
+                                                .putList(
+                                                    "required",
+                                                    singletonList(Document.fromString("location")))
+                                                .build())
+                                        .build())
+                                .build())
+                        .build()))
+            .build();
+
+    List<Document> messages = new ArrayList<>();
+    messages.add(
+        Document.mapBuilder()
+            .putString("role", "user")
+            .putList(
+                "content",
+                singletonList(
+                    Document.mapBuilder()
+                        .putString(
+                            "text", "What is the weather in Seattle and San Francisco today?")
+                        .build()))
+            .build());
+
+    Document requestPayload0 =
+        Document.mapBuilder()
+            .putList("messages", messages)
+            .putDocument("toolConfig", toolConfig)
+            .build();
+
+    SdkJsonGenerator generator0 = new SdkJsonGenerator(new JsonFactory(), "application/json");
+    software.amazon.awssdk.protocols.json.internal.marshall.DocumentTypeJsonMarshaller marshaller0 =
```

Comment:
does this need to be qualified?

Reply:
Thanks! Not sure why IntelliJ did that when copy-pasting in